<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Replace</class>
 <widget class="QDialog" name="Replace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_find">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>替换</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_find">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查找内容：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_find">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>区分大小写</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_up">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>向上</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_down">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>向下</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_replace">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>替换为：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_replace">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
